app/controllers/finders_controller.rb

```diff
@@ -64,16 +64,9 @@ def raw_finder
   end
 
   def fetch_breadcrumbs
-    parent_content_item = {}
-    unless params.dig("parent_path").to_s.empty?
-      begin
-        parent_content_item = Services.content_store.content_item(params["parent_path"])
-      rescue GdsApi::HTTPNotFound
-        #parent_path is user input so we don't mind if it's bad
-        GovukStatsd.increment("breadcrumb.parent_path_not_found")
-      end
-    end
-    FinderBreadcrumbsPresenter.new(parent_content_item, @content_item)
+    parent_slug = params["parent"]
+    org_info = org_registry[parent_slug] if parent_slug.present?
+    FinderBreadcrumbsPresenter.new(org_info, @content_item)
```

Comment:
What will `FinderBreadcrumbsPresenter` do when `org_info` is nil?

Reply:
It turns out I'd deleted that bit when I rebased.  Re-added, thanks